auto_submit/lib/git/git_access_method.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// There are two ways to clone the repository which will be configurable in the
+/// repository configuration in .github.
+enum GitAccessMethod {
```

Comment:
Can we consolidate these small dart files? Maybe a common git utility file?

Reply:
Yeah I can add it to the gitcli class